scripts/coverage/coverage_capture.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+# Copyright (C) 2024 Intel Corporation
+# Under the Apache License v2.0 with LLVM Exceptions. See LICENSE.TXT.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+set -e
+
+[ "$1" != "" ] && OUTPUT_NAME="$1" || OUTPUT_NAME="total_coverage"
```

Comment:
I guess you could add a comment saying, that if this script is run with no params it would just calculate a coverage for a single build (in comparison to CI, where we merge coverage from various builds/jobs)

Reply:
Added the comment:
```shell
# This script calculates coverage for a single build
```